### ISSUE TITLE
Conduit dashboard fails with "NetworkError" in Firefox nightly on master

### ISSUE THREAD

**Issue by hawkw** [runconduit/conduit#930]
While testing some unrelated changes, I came across what appears to be a bug in the web dashboard. The current master (25d8b22b5ccb07023de24d4bc1ba1c28b4ae19f7) fails to fully load the page in Firefox 62.0a1 (2018-05-09) , displaying the following error message:
```
Error getting data from server: NetworkError when attempting to fetch resource.
```

However, it loads successfully in Safari 11.1 (12605.1.33.1.4).

The on the branch I was testing (PR #916), the dashboard loads correctly in both Firefox and Safari. This branch diverged from `master` after commit 7f16079f644612f9ff548cf11b39e969c11a399a, so I suspect the bug was added somewhere between 7f16079f644612f9ff548cf11b39e969c11a399a and 25d8b22b5ccb07023de24d4bc1ba1c28b4ae19f7.

/cc @rmars

**Comment by rmars** [comment]
I've been unable to reproduce this with Firefox `60.0.1 (64-bit)` or `61.0b5 (64-bit)` on the latest master :(

Is this still happening for you? If you get it to happen again, I'd like to take a look.

**Comment by hawkw** [comment]
It's still happening for me with Firefox `62.0a1 (2018-05-15) (64-bit)`.

**Comment by rmars** [comment]
I downloaded Nightly (62.0a1 (2018-05-17) (64-bit)) and `conduit dashboard` works for me :(
I am running conduit in minikube, on a branch off master from yesterday (a1b56846b227daa686d57caa37ffb40ac0dde40c)

**Comment by rmars** [comment]
Our current theory is that this is #970 (Adblocker)

**Comment by hawkw** [comment]
I think this can be closed as a duplicate of #970.
